exasol_integration_test_docker_environment/lib/base/task_state.py

```diff
@@ -4,7 +4,9 @@
 class TaskState(Enum):
     NONE = 0,
     INIT = 1,
+    AFTER_INIT =6,
     RUN = 2,
     FINISHED = 3,
     CLEANUP = 4,
     CLEANED = 5
+    ERROR = 7
```

Comment:
```suggestion
    NONE = 0,
    INIT = 1,
    AFTER_INIT =2,
    RUN = 3,
    FINISHED = 4,
    CLEANUP = 5,
    CLEANED = 6,
    ERROR = 7
```

Reply:
currently, this cannot work, missing comma after CLEANED = 5,

Reply:
it seems the commas are not necessary, because I don't get syntax errors